Makefile

```diff
@@ -34,8 +34,7 @@ run: build
 	docker run -e MYHOSTNAME=localhost -e PLAIN_AUTH $(IMAGE_REPOSITORY)
 
 test: build
-	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf-env-set
-	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf -l *.py
+	MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} make -C tests test
```

Comment:
I thought that the metadata file would take care of this horrendously long command line.

Reply:
partially could, but there is more things what are generated by this Makefile dynamically:
* what can be handled by metadata (both ot them can be ommited also now):
  * MODULE=docker
  * DOCKERFILE="../$(DOCKERFILE)"
* what cannot be handled (set by some tooling inside makefile):
  * URL="docker=${IMAGE_REPOSITORY}"
  * VERSION=${VERSION}
  * DISTRO=${DISTRO}

Reply:
okay then